src/VoteHandler.jsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from "react";
+import { upvoteArticleVotes, downvoteArticleVotes } from "../api";
+
+function VoteHandler(props){
+
+    const {articleId} = props;
+
+    const [voteCount, setVoteCount] = useState(0)
+
+    const [error, setError] = useState(null)
+
+    useEffect(() => {
```

Comment:
We shouldn't need a use effect here. This is just setting the state to be the same value again